### PR TITLE
km dcp 13 release updates

### PR DESCRIPTION
**Pull request by kayleemathews**

### CODE REVIEW

File: content/community-update/community-update/dcp-updates.md [hunk 1]
```diff
@@ -293,12 +352,12 @@ Managed access raw data is available from the [Synapse database](https://www.syn
...
 * Raw data and contributor-generated matrices are available for 15 new projects. These single-cell data are derived from:
 * Single-cell and single-nucleus
-* 10x 3’ v2 and v3 3’ and 10x 5’ chemistry, Fluidigm C1, Smart-seq2, and sci-CAR ATAC sequencing technologies
+* 10x 3' v2 and v3 3' and 10x 5' chemistry, Fluidigm C1, Smart-seq2, and sci-CAR ATAC sequencing technologies
 * Adipose tissue, blood, bone marrow, CSF, heart, kidney, lung, oral cavity, ovary, pancreas, prostate, skeletal muscle, testis, and umbilical vein
 * Disease states including COVID-19 infection, intracranial hypertension, multiple sclerosis, and renal cell carcinoma
 * Developmental stages including fetal, child, adolescent, and adult
 * Standardized data, including aligned BAMs and cell-by-gene count matrices (Loom format), are available for 8 additional projects with data derived from:
-* 10x v2 and v3 3’ sequencing technologies
+* 10x v2 and v3 3' sequencing technologies
```
Comment (ekiernan) [inline content/community-update/community-update/dcp-updates.md:360]: @kayleemathews What's going on with the single quotes here? I pulled the main branch and they looked normal.

File: content/community-update/community-update/dcp-updates.md [hunk 2]
```diff
@@ -6,6 +6,65 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### January 24, 2022
+
+### Release Highlights
+* We've created a new [slack channel](https://humancellatlas.slack.com/archives/C02TM2SDVM2) for the DCP community! Join the channel for announcements, updates, help, and an opportunity to connect with other users of the DCP.
```
Comment (ekiernan) [inline content/community-update/community-update/dcp-updates.md]:
```suggestion
* We've created a new [slack channel](https://humancellatlas.slack.com/archives/C02TM2SDVM2) for the DCP community! Join the channel, say hello, and follow for the latest announcements, updates, and opportunities to connect with fellow DCP users.
```

File: content/community-update/community-update/dcp-updates.md (hunk 1 above)
Comment (kayleemathews) [reply content/community-update/community-update/dcp-updates.md:360]: I may have accidentally updated them with Grammarly.

**Review by ekiernan — APPROVED** [review]
LGTM! Thanks for making the release notes @kayleemathews. I had one suggestion and one question. The release isn't live yet, so I wasn't able to confirm the links yet.

File: content/community-update/community-update/dcp-updates.md (hunk 1 above)
Comment (kayleemathews) [reply content/community-update/community-update/dcp-updates.md:360]: It doesn't like when there are both straight and curly apostrophes in a doc.

Comment (ekiernan) [reply content/community-update/community-update/dcp-updates.md:360]: oh! That makes sense. Grammarly!! (shakes fist) Thank you!

Comment (kayleemathews) [reply content/community-update/community-update/dcp-updates.md:360]: Are they okay like that or should I revert those changes?

Comment (ekiernan) [reply content/community-update/community-update/dcp-updates.md:360]: I don't see a problem leaving them. It didn't appear to affect any title searches, so I think it's ok.

**Comment by kayleemathews** [comment]
@ekiernan thanks for the review and suggestion!

**Review by NoopDog — APPROVED** [review]

**Pull request merged**